src/lib/swapper/swappers/OsmosisSwapper/OsmosisSwapper.ts

```diff
@@ -427,14 +426,30 @@ export class OsmosisSwapper implements Swapper<ChainId> {
     /** Execute the swap on Osmosis DEX */
     const osmoAddress = sellAssetIsOnOsmosisNetwork ? sellAddress : receiveAddress
     const cosmosAddress = sellAssetIsOnOsmosisNetwork ? receiveAddress : sellAddress
+
+    /** At the current time, only OSMO<->ATOM swaps are supported, so this is fine.
+     * In the future, as more Osmosis network assets are added, the buy asset should
+     * be used as the fee asset automatically. See the whitelist of supported fee assets here:
+     * https://github.com/osmosis-labs/osmosis/blob/04026675f75ca065fb89f965ab2d33c9840c965a/app/upgrades/v5/whitelist_feetokens.go
+     */
+    const swapFeeData = await (sellAssetIsOnOsmosisNetwork
+      ? osmosisAdapter.getFeeData({})
+      : cosmosAdapter.getFeeData({}))
+
+    const feeDenom = sellAssetIsOnOsmosisNetwork
+      ? 'uosmo'
+      : atomOnOsmosisAssetId.split('/')[1].replace(/:/g, '/')
```

Comment:
Either:

```suggestion
      // ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2
      : atomOnOsmosisAssetId.split('/')[1].replace(/:/g, '/')
```

Or:

```suggestion
      : 'ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2'
```

Reply:
It's a constant either way - which is why I've at least adding a comment to make it clear what's the output here for grep purposes